core/sqf/src/tm/tm.cpp

```diff
@@ -460,9 +460,13 @@ void tm_process_req_requestregioninfo(CTmTxMessage * pp_msg)
        TM_Txid_legacy lv_transid;
    } u;
 
-   char tname[2000], ername[50], rname[100], offline[20], regid[200], hostname[200], port[100];
+   char tname[2000];
+   tname[299] = '\0';
```

Comment:
Odd that we'd set just element 299 of a 2000-byte buffer to null. But I see that this is the way it was before.